src/components/typography/typography.module.scss

```diff
@@ -0,0 +1,37 @@
+@font-face {
+  font-family: "Source Sans Pro";
```

Comment:
I think, this could be also be `font-family: $font-family-source-sans-pro, "Helvetica Neue", "Helvetica",
    "Arial";` ?